src/validation/Validator.ts

```diff
@@ -717,9 +717,10 @@ export class Validator {
     /**
      * Checks if the string is a valid ISO 8601 date.
      * If given value is not a string, then it returns false.
+     * Use the option strict = true for additional checks for a valid date, e.g. invalidates dates like 2019-02-29.
      */
-    isISO8601(value: unknown): boolean {
-        return typeof value === "string" && this.validatorJs.isISO8601(value);
+    isISO8601(value: unknown, strict?: boolean): boolean {
```

Comment:
Please use `options?: ValidatorJS.IsISO8601Options` instead of only one `strict` argument. It cover cases when validatorjs add new options in future.

Reply:
You're right, I also thought about this in the morning)